documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -44,6 +44,15 @@ These strategies can be configured as a sequence.
 The first strategy attempts to compare two partition reassignments using its internal logic. 
 If the reassignments are equivalent, then it passes them to the next strategy in the sequence to decide the order, and so on.
 
+[discrete]
+== Intra-broker disk balancing
+
+Moving a large amount of data between disks on the same broker has less impact than between separate brokers
+If you are running a Kafka deployment that uses JBOD storage, Cruise Control can balance partitions between disks on the same broker. 
```

Comment:
I think you need to distinguish two different things => use JBOD storage versus has multiple disks. We recommend everyone to use JBOD storage (as a Strimzi storage type) even if they use it with single disk because they can add more later. So to which part does this apply? I assume users using JBOD storage with a single disk should be fine? I think we need to clarify this both here and in other places in the docs / APi docs in this PR.

Reply:
JBOD storage with a single disk should be fine, I assume if a user executes an intra-broker disk balancing while using one disk will just result in a proposal with 0 partition movements but I will test it out just to make sure and get back to you.

Reply:
Yeah, i would expect it to be fine since Cruise Cotnrol does not know different. But the docs are not clear about this. I think you need to always say somehting like _JBOD with multiple disks per broker_ or soemthing like that,

Reply:
_NOTE: If you are using JBOD storage with a single disk, intra-broker disk balancing between disks is not applicable._  Note after the next sentence

Reply:
I would not use the wording "is not applicable" (at least as it sounds as non native speaker). I would specify, as Kyle mentioned, that it would result in a proposal with 0 partition movements. Using "is not applicable" sounds to me as it doesn't work at all and could be confusing imho.